packages/rx_bloc_cli/mason_templates/bricks/lib_auth/__brick__/lib/lib_auth/services/user_account_service.dart

```diff
@@ -19,8 +21,6 @@ class UserAccountService {
   final PushNotificationRepository _pushSubscriptionRepository;
   final PermissionsService _permissionsService;
 
-  bool _logoutLocked = false;
```

Comment:
I believe we will need this prop. @PVGeorgievPrime, can you please confirm that ?